app/components/submit_button_component.rb

```diff
@@ -0,0 +1,15 @@
+class SubmitButtonComponent < ApplicationComponent
+  attr_reader :text, :disable_with, :button_clasess
+
+  def initialize(text:, disable_with: nil, button_classes: nil)
+    @text = text
+    @disable_with = disable_with
+    @button_clasess = button_clasess || default_button_clasess
```

Comment:
I think `button_classes` should be additive, not destructive. Said another way, it should append `button_classes` to `default_button_classes`. I'm also not sure I want `ml-3` as the default. But you were in the code – was every submit button using that class?

Reply:
Got it!